pycbc/inference/likelihood.py

```diff
@@ -718,6 +723,21 @@ def loglr(self, **params):
                 # if the waveform terminates before the filtering low frequency
                 # cutoff, there is nothing to filter, so just go onto the next
                 continue
+            if self.calib:
```

Comment:
All of this below should make its way upstream to ``pycbc_inference``. And just pass the likelihood class ``recailbrator`.

Don't initialize the class over and over each iteration.